keyboards/keebio/viterbi/keymaps/vosechu/keymap.c

```diff
@@ -0,0 +1,116 @@
+#include QMK_KEYBOARD_H
+#include "vosechu.h"
+
+extern keymap_config_t keymap_config;
```

Comment:
This isn't actually needed, as it's included already.

```suggestion
```

Reply:
Whew! I'm glad. I hadn't seen this before, so I just accepted it as dark voodoo that had to be kept. Really glad you're keeping an eye on PRs!